.changeset/heavy-gorillas-march.md

```diff
@@ -0,0 +1,5 @@
+---
+"ts-essentials": patch
+---
+
+`MarkOptional<Type, Keys extends keyof Type>` is now assignable to `Partial<Type>`
```

Comment:
I suggest simplifying the expression, because we don't mention it in README and other docs either:

```suggestion
`MarkOptional<Type, Keys>` is now assignable to `Partial<Type>`
```

Reply:
Ok cool, updated it in [this commit](https://github.com/ts-essentials/ts-essentials/pull/418/commits/b0e6455cc0a548279e33d4416979106648a71eca).